catalog.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en-ru">
```

Comment:
```suggestion
<html lang="ru">
```
Атрибут lang (https://www.w3.org/TR/html52/dom.html#the-lang-and-xmllang-attributes - можно ограничиться первым абзацем).